packages/toast/api.js

```diff
@@ -1,161 +1,58 @@
 
-export function initToasts() {
-  if (!windowExists) return;
-  if (!window.fabricToasts) window.fabricToasts = { toasts: [] };
-  if (document && !document.querySelector('f-toast-container')) {
-    class Container extends HTMLElement {
-      constructor() {
-        super();
-
-        const fabricStylesTemplate = document.createElement('template');
-        fabricStylesTemplate.innerHTML = `
-                <style>:host { display: block; }</style>
-                <link
-                    rel="stylesheet"
-                    type="text/css"
-                    href="https://assets.finn.no/pkg/@fabric-ds/css/v0/fabric.min.css"
-                />
-            `;
-
-        this.attachShadow({ mode: 'open' }).appendChild(
-          fabricStylesTemplate.content,
-        );
-      }
-
-      connectedCallback() {
-        const container = document.createElement('aside');
-        container.className = c.toasterContainer;
-
-        const div = document.createElement('div');
-        div.id = 'f-toast-container-list';
-        div.className = `${c.toaster} relative`;
-
-        container.appendChild(div);
-
-        this.shadowRoot.appendChild(container);
-      }
-    }
-    customElements.define('f-toast-container', Container);
-
-    const container = document.createElement('f-toast-container');
-    document.querySelector('body').appendChild(container);
-  }
-}
-
 /**
  * Toast helper function options
  * @typedef    {Object}                               ToastOptions
  * @property   {(number|string)}                      [id]        Custom identifier
  * @property   {('success'|'error'|'warning'|'info')} [type]      Type of alert
  * @property   {String}                               [text]      The toast message. Only needed when updating text on existing toast
  * @property   {(number|string)}                      [duration]  Duration of toast in milliseconds Set to 0 to disable auto-removal
- * @property   {Boolean}                              [canClose]  Can toast be dismissed?
- * @property   {Function}                             [onClose]   Dismiss handler
+ * @property   {Boolean}                              [canclose]  Can toast be dismissed?
  */
 
 /**
- * Create a new toast
- * @param {String}       message Message
- * @param {ToastOptions} [options] Toast options
+ * Creates a new toast
+ * @param {String}        message  Message
+ * @param {ToastOptions?} options  Toast options
  * @returns {ToastOptions} Toast details
  */
 export function toast(message, options) {
   if (!windowExists) return;
-  const toasts = window.fabricToasts;
-  const container = document.querySelector('f-toast-container');
-  const list = container.shadowRoot.getElementById('f-toast-container-list');
-
-  if (!container && customElements.get('f-toast') && toasts) {
-    throw new Error(
-      'No toast container found. Make sure to call initToasts() first.',
-    );
-  }
+  const toast = FabricToastContainer.init();
```

Comment:
The init method ensures that no matter how many times its called, only 1 instance of the toast container is ever added to the page. This should make it safe for multiple podlets within a page to use the apis with fear of creating more toast containers